docs/kit/adoption-view/page_adoption-view.md

```diff
@@ -36,13 +62,113 @@ The main difference between the EDC and the previous connectors of the [IDSA][id
+- An interface to the Identity Provider service, currently the [Managed identity Wallet][miw-url]. This central service provides the identity and the corresponding authentication of the participants in the data exchange. Decentralized solutions will also be supported in the future, following [Gaia-X]'s[gaiax-url] principles and implementing self sovereign identity.
 - The provision of possible offers (contract offering) which, on the one hand, stipulates the data offered and the associated terms of use (policies) in corresponding contracts.
 - An interface for manual selection of data and associated contract offers.
 - The actual data transfer via the data plane extension
 - The connection of software systems on the customer and provider side
 
+## Additional Resources
+
+### Catena-X Standards
+
+The Connector KIT builds on the [Catena-X Standards][Catena-X-Standards-url]. Every data consumer and provider in Catena-X is required to comply with these standards. The [EDC Connector][edc-url] builds on the following standards:
+
+- [CX - 0002 Digital Twins in Catena - X][CX0002]
+- [CX - 0003 BAMM Aspect Meta Model][CX0003]
+- [CX - 0008 Relevant standards for conformity assessments][CX0008]
+- [CX - 0018 Eclipse Data Space Connector (EDC)][CX0018]
+- [CX - 0044 ECLASS][CX044]
+- [CX - 0047 DCM Data Model Material Demand & Capacity Group][CX0047]
+- [CX - 0059 Triangle Behavioral Twin Endurance Predictor][CX0059]
+
+### Terminology
+
+![domain-model](images/domain-model.png)
+
+> The shown picture illustrates only a generic view of the Domain Model and is not intended to show all aspects of the project.
+
+#### Extensions
+
+There are different extenions for the Connector, e.g. for the Data Plane. This enables various tranfer modes like httpData or via blob-storage.
+
+#### Data Plane
+
+The Data Plane handles several forms of actual data exchange by utilizing various extensions.
+
+#### Control Plane
+
+The Control Plane handles meta data exchange with other components and Connectors, as well as transfer of access tokens
+
+#### Data Assets
+
+Data Sources (databases, files, cache information, etc.) are connected to the Connector and are represented by Data Assets. For each asset, a [`DataAddress`](#data-address) needs to be resolvable.
+
+#### Data address
+
+A data address is a pointer into the physical storage location where an asset will be stored.
+
+#### Policy Definition
+
+A standardized set of policies can be used to define actions regarding access to and usage of assets. These actions can be limited further by constraints (temporal or spatial) and duties ("e.g. deletion of the data after 30 days").
+
+#### Contract Definition
+
+By combining Assets and Policies, Contracts for data offerings are defined. These Contracts need to be accepted by consuming participants (Connectors) for the data exchange to take place.
+
+#### Contract offer
+
+The contract offer is a dynamic representation of the [`ContractDefinition`](#contract-definition)
```

Comment:
I'm not sure I understand this definition :D

Reply:
I updated it. Might have to adapt after tomorrows release of standards and guidelines again.